duti.c

```diff
@@ -3,147 +3,145 @@
 #include <ApplicationServices/ApplicationServices.h>
 #include <CoreFoundation/CoreFoundation.h>
 
-#include <sys/types.h>
-#include <sys/stat.h>
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
 #include <unistd.h>
 
 #include "handler.h"
 
-extern char		*duti_version;
-extern int		nroles;
-int			verbose = 0;
+extern char *duti_version;
+extern int nroles;
+int verbose = 0;
 
-struct roles		rtm[] = {
-    { "none",   kLSRolesNone },
-    { "viewer", kLSRolesViewer },
-    { "editor", kLSRolesEditor },
-    { "shell",  kLSRolesShell },
-    { "all",    kLSRolesAll },
+struct roles rtm[] = {
+    {"none", kLSRolesNone},     {"viewer", kLSRolesViewer},
+    {"editor", kLSRolesEditor}, {"shell", kLSRolesShell},
+    {"all", kLSRolesAll},
 };
 
-    int
-main( int ac, char *av[] )
-{
-    struct stat		st;
-    int			c, err = 0;
-    int			set = 0;
-    int			( *handler_f )( char * );
-    char		*path = NULL;
-    char		*p;
-
-    extern int		optind;
-    extern char		*optarg;
-
-    while (( c = getopt( ac, av, "d:e:l:hsu:Vvx:" )) != -1 ) {
-	switch ( c ) {
-	case 'd':	/* show default handler for UTI */
-	    return( uti_handler_show( optarg, 0 ));
-
-	case 'e':	/* UTI declarations for extension */
-		return( duti_utis_for_extension( optarg ));
-
-	case 'h':	/* help */
-	default:
-	    err++;
-	    break;
-
-	case 'l':	/* list all handlers for UTI */
-	    return( uti_handler_show( optarg, 1 ));
-
-	case 's':	/* set handler */
-	    set = 1;
-	    break;
-
-	case 'u':	/* UTI declarations */
-		return( duti_utis( optarg ));
-
-	case 'V':	/* version */
-	    printf( "%s\n", duti_version );
-	    exit( 0 );
-
-	case 'v':	/* verbose */
-	    verbose = 1;
-	    break;
-
-	case 'x':	/* info for extension */
-	    return( duti_default_app_for_extension( optarg ));
-	}
-    }
+int main(int ac, char *av[]) {
+  struct stat st;
+  int c, err = 0;
+  int set = 0;
```

Comment:
Moved to all one line. This PR is focused purely on style, I'll look at refactoring in a future PR.